SitecoreDeployer/buildtask/functions-module-v1.psm1

```diff
@@ -267,10 +267,10 @@ function SetWebAppRestrictions {
     #get current web app config
     if ($isSlot) {
         Write-Verbose "We are working with slot"
-        $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites/slots/config -ResourceName $webAppInstanceName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)
+        $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites/slots/config -Name $webAppInstanceName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)
     } else {
         Write-Verbose "We are working with web app"
-        $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites/config -ResourceName $webAppInstanceName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)
+        $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites/config -Name $webAppInstanceName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)

```

Comment:
Please, leave ```ResourceName``` here to maintain backward compatibility with older powershell versions (see https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/get-azurermresource?view=azurermps-6.13.0 -  ```ResourceName``` is alias for ```-Name```, while in Powershell v.5 and lower, there no ```-Name``` at all